sklearn/datasets/openml.py

```diff
@@ -399,6 +436,10 @@ def fetch_openml(name=None, version='active', data_id=None, data_home=None,
         If True, returns ``(data, target)`` instead of a Bunch object. See
         below for more information about the `data` and `target` objects.
 
+    verify_checksum : boolean, default=True
+        Whether or not to validate that the dataset file's MD5 hash matches the
+        data set description's expected checksum.
```

Comment:
Maybe note that "Verification occurs when fetching the data, so if cache=True, the data will be presumed valid on subsequent calls."